### ISSUE TITLE
Missing flow binaries in local tests.

### ISSUE THREAD

**Issue by AndrewSouthpaw** [flowtype/flow-typed#1759]
I'm having an issue getting my tests to work correctly. My current [PR build](https://travis-ci.org/flowtype/flow-typed/jobs/332878174) fails on Flow 0.62 and 0.63, but I can't get them to download and run locally:

```
$ ./run_def_tests.sh ramda_v0.x.x

...snip...

Fetching all Flow binaries...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.52.0)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.53.0)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.53.1)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.54.0)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.54.1)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.55.0)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.56.0)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.57.1)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.57.2)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.57.3)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.58.0)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.59.0)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.60.0)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.60.1)...
Testing ramda_v0.x.x/flow_v0.49.x- (flow-v0.61.0)... <--- no 0.62 or 0.63!

All tests passed!
```

**Comment by KenanY** [comment]
Mac or Linux?

**Comment by GAntoine** [comment]
Yeah I noticed that a while ago, wrote a fix at #1630. I'm just waiting on the next opportunity to deploy a patch to production.

**Comment by AndrewSouthpaw** [comment]
Hum. I'm confused, I have that fix in my codebase... meaning what's happening locally is correct?

**Comment by GAntoine** [comment]
Wait never mind, that PR fixes another issue with windows systems. Looks like this is a legit bug, thanks for bringing it up. I'm experiencing the same issue on my end, so I'll take a look at the code that updates the cache when I get some time tomorrow.

If you're up for it and can look into it as well, I'd appreciate the help. 💪

**Comment by KenanY** [comment]
@AndrewSouthpaw Could you try the patch in this comment? https://github.com/flowtype/flow-typed/issues/1724#issuecomment-359816295

**Comment by KenanY** [comment]
Alternatively, this could be resolved now that https://github.com/facebook/flow/issues/5709 is resolved (it's hard to tell when the OSX build became available).

**Comment by AndrewSouthpaw** [comment]
I'd be happy to help out. 

It might make sense to pair on it, since I've never walked around in the codebase before. If that interests you, feel free to ping me on Google Hangouts (andrewsouthpaw@gmail.com) or Skype (andrewsouthpaw) and we can find a time to hash it out. 🙌

**Comment by GAntoine** [comment]
@AndrewSouthpaw So after some digging, I think I've found the first clue. Since `0.63.1`, the releases contain `libflowparser-osx-v0.63.1.zip`, which causes `runTests.js:157` to throw (silently?). Since it tries to install the latest one first (0.64.x), it trips, fails and just runs the specs without caching the latest flow versions.

I've reached out to calebmer for some insight. Once he gets back to me, a fix should be as simple as working around the presence of multiple packages/OS in a release.

I'd be down to pair on this, it's great to have more people around familiar with the code base. Unless something comes up, I'll ping you this evening on hangouts.
